scss/components/_modal.scss

```diff
@@ -31,6 +31,7 @@
 
     // Content
     article {
+      $close-selector: if($enable-classes, ".close, a[rel='prev']", "a[rel='prev']");
```

Comment:
I was about to comment here about adding  `.close, a[rel='prev']`, but then I just saw your force-push

Reply:
😅